vagrant/provisioning/roles/alfresco/tasks/main.yml

```diff
@@ -228,6 +228,14 @@
     dest: "{{ root_folder }}/app/alfresco/shared/classes/alfresco/extension/audit"
   when: enable_alfresco_sync | bool
 
+- name: unarchive alfresco-pdf-renderer
+  become: yes
+  become_user: alfresco
+  unarchive: 
+    remote_src: yes
+    src: "{{ root_folder }}/install/alfresco/alfresco-content-services-community-distribution-{{ alfresco_content_services_version }}/alfresco-pdf-renderer/alfresco-pdf-renderer-1.1-linux.tgz"
```

Comment:
I've noticed that this is a CE install. What's the plan for EE version?

Reply:
I fixed these tickets first because they're easy, and because they apply both to CE and EE Alfresco.  Now I have to actually address EE Alfresco, which means uploading the artifacts to our SFTP site, and reorganizing the Alfresco role into several roles, like Pentaho (first a common setup role, then either CE or EE specific role, then a common post-install role).